pyproject.toml

```diff
@@ -204,6 +208,13 @@ ignore = [
     "B017",    #`pytest.raises(Exception)` should be considered evil
     "RUF005",  # Consider "...."instead of concatenation
     "RUF012",  # Mutable class attributes should be annotated with "typing.ClassVar"
+    "TC001",   # Type-checking block
+    "TC002",   # Type-checking block
+    "TC003",   # Type-checking block
+    "TC004",   # Type-checking block
```

Comment:
Do we plan to enable these in the future? I thought it was the main benefit of `flake8-type-checking`?

Reply:
Yes and no. We should discuss this before enabling it because I wouldn't say I like the actual implications and syntax.